WordPress/Classes/ViewRelated/Me/App Settings/Privacy Settings/PrivacySettingsViewController.swift

```diff
@@ -7,15 +7,20 @@ class PrivacySettingsViewController: UITableViewController {
 
     // MARK: - Dependencies
 
-    private let analyticsTracker = PrivacySettingsAnalyticsTracker()
+    private let analyticsTracker: PrivacySettingsAnalyticsTracking
 
     // MARK: - Init
 
-    override init(style: UITableView.Style) {
+    init(style: UITableView.Style, analyticsTracker: PrivacySettingsAnalyticsTracking) {
+        self.analyticsTracker = analyticsTracker
         super.init(style: style)
         navigationItem.title = NSLocalizedString("Privacy Settings", comment: "Privacy Settings Title")
     }
 
+    override convenience init(style: UITableView.Style) {
+        self.init(style: style, analyticsTracker: PrivacySettingsAnalyticsTracker())
+    }
```

Comment:
I don't know if the code is self-explanatory or not 🤔 

If it's not clear, we need to make `analyticsTracker: PrivacySettingsAnalyticsTracking` injectable but we can only do it with a custom `init`. Adding a custom `init` though, might leave the door open for misusing the super class ones, which is why it's overriddend here with the default parameter.

Note that it shouldn't be possible to have both `init(style: UITableView.Style, analyticsTracker: PrivacySettingsAnalyticsTracking = PrivacySettingsAnalyticsTracker())` and `init(style: UITableView.Style)` because the compiler would know what to resolve init(style: <value>)` to.

Reply:
Thanks for the explanation and making `analyticsTracker` injectable. The code is clear. 👍